sch

```diff
@@ -15,6 +15,4 @@ if [ ! "$(grep -q "$search" < ~/.config/search/search_history)" ]; then
 fi
 
 # Open browser if search query provided
-if [ -n "$search" ] && [ "$search" != "" ]; then
-    opnbrow "$1" "$url$(encode "$search")"
-fi
+[ -n "$search" ] && opnbrow "$1" "$url$(encode "$search")"
```

Comment:
My approach would be the same, since the logic here was essentially doing the same thing twice. In other languages, though, this may be a sensible approach, but not here, AFAIK.